Jenkinsfile

```diff
@@ -3,7 +3,7 @@ properties(
     parameters(
       [
         [$class: 'ValidatingStringParameterDefinition',
```

Comment:
This block, that addes the ARCHES parameter, should be deleted since it's no longer used.